src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -172,36 +203,39 @@ public enum SecurityContextSource {
-    [System.ObsoleteAttribute]
-    public static System.Collections.IEnumerator PolicyHierarchy() { throw null; }
-    [System.ObsoleteAttribute]
-    public static System.Security.PermissionSet ResolvePolicy(System.Security.Policy.Evidence evidence) { throw null; }
-    [System.ObsoleteAttribute]
-    public static System.Security.PermissionSet ResolvePolicy(System.Security.Policy.Evidence evidence, System.Security.PermissionSet reqdPset, System.Security.PermissionSet optPset, System.Security.PermissionSet denyPset, out System.Security.PermissionSet denied) { throw null; }
-    [System.ObsoleteAttribute]
-    public static System.Security.PermissionSet ResolvePolicy(System.Security.Policy.Evidence[] evidences) { throw null; }
-    [System.ObsoleteAttribute]
-    public static System.Collections.IEnumerator ResolvePolicyGroups(System.Security.Policy.Evidence evidence) { throw null; }
-    [System.ObsoleteAttribute]
-    public static System.Security.PermissionSet ResolveSystemPolicy(System.Security.Policy.Evidence evidence) { throw null; }
-    [System.ObsoleteAttribute]
-    public static void SavePolicy() { }
-    [System.ObsoleteAttribute]
-    public static void SavePolicyLevel(System.Security.Policy.PolicyLevel level) { }
-  }
-  public abstract partial class SecurityState {
+    public static partial class SecurityManager
+    {
+        [System.ObsoleteAttribute("Because execution permission checks can no longer be turned off, the CheckExecutionRights property no longer has any effect.")]
+        public static bool CheckExecutionRights { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
+        [System.ObsoleteAttribute("Because security can no longer be turned off, the SecurityEnabled property no longer has any effect.")]
+        public static bool SecurityEnabled { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
+        public static bool CurrentThreadRequiresSecurityContextCapture() { throw null; }
+        public static System.Security.PermissionSet GetStandardSandbox(System.Security.Policy.Evidence evidence) { throw null; }
+        public static void GetZoneAndOrigin(out System.Collections.ArrayList zone, out System.Collections.ArrayList origin) { zone = default(System.Collections.ArrayList); origin = default(System.Collections.ArrayList); }
+        [System.ObsoleteAttribute("IsGranted is obsolete and will be removed in a future release of the .NET Framework.  Please use the PermissionSet property of either AppDomain or Assembly instead.")]
+        public static bool IsGranted(System.Security.IPermission perm) { throw null; }
+        [System.ObsoleteAttribute("This method is obsolete and will be removed in a future release of the .NET Framework. See http://go.microsoft.com/fwlink/?LinkID=155570 for more information.")]
```

Comment:
I think you can just eliminate all the ObsoleteAttribute text. We barely need the attributes really. We don't expect anyone to write new code against this because it isn't actually implemented. Kind of odd to warn something will be removed in future if we didn't implement it already.

So just [System.ObsoleteAttribute] is fine.